src/PKSim.Core/Snapshots/Mappers/QuantityInfoMapper.cs

```diff
@@ -14,15 +15,16 @@ public override Task<SnapshotQuantityInfo> MapToSnapshot(ModelQuantityInfo quant
          {
             snapshot.OrderIndex = SnapshotValueFor(quantityInfo.OrderIndex);
             snapshot.Path = SnapshotValueFor(quantityInfo.PathAsString);
-            snapshot.Type = SnapshotValueFor(quantityInfo.Type, QuantityType.Undefined);
+            snapshot.Type = quantityInfo.Type != QuantityType.Undefined ? quantityInfo.Type.ToString() : null;
```

Comment:
Json Schema does nto support validation of Flag Enum (E.g. Drug|Observer) and would fail.
I am starting to wonder if we should not disable validation with schema especially considering the fact that those json files are created by PKSIm and are not coming from 3rd party tools

Reply:
Unless we start e.g. exchanging compounds via snapshots...

Reply:
>Or we could turn out schema validation for compound only when we are ready.

Sounds reasonable for me